src/bentoml/_internal/context.py

```diff
@@ -73,7 +73,7 @@ def mutablecopy(self) -> Metadata:
 
 
 @attr.define
-class InferenceApiContext:
+class Context:  # NOTE: This is the InferenceApiContext
```

Comment:
Why the rename?

Reply:
For the LazyModule import attribute to work properly